### ISSUE TITLE
lldb bindings indicate conflict in option names

### ISSUE THREAD

**Issue by compnerd** [swiftlang/llvm-project#5884]
```
error: option[26] --bind-generic-types has a short option 0x1 that conflicts with option[9] --show-tags, short option won't be used for --bind-generic-typesn
```

There seem to be potentially two separate issues:
- conflicting short name for `--show-tags` and `--bind-generic-types`
- missing `\` on the `\n`

CC: @JDevlieghere @kastiglione

**Comment by JDevlieghere** [comment]
I fixed the newline in 7174568c9ccf2959b95563252c1bdbf59c5746c7.

**Comment by augusto2112** [comment]
Fix: https://github.com/apple/llvm-project/pull/6124
